### PR TITLE
separate messages file.

### PR DESCRIPTION
**Pull request by TharunCBT**
- Added a separate messages.py file to get the success and error messages from it.
- Removed commented out code and some empty extra lines.
- Changed the toast message in additional and unavailable slot.

### CODE REVIEW

**Review by rugvedcbt — APPROVED** [review]

File: backend/base/views.py [hunk 1]
```diff
@@ -1595,7 +1461,7 @@ def form_valid(self, form):
             location = OrganizationLocation.objects.get(pk=pk)
             form.instance.location = location
             response = super().form_valid(form)
-            messages.success(self.request, 'Created Unavailable slot successfully !!!')
+            messages.success(self.request, SUCCESS_MESSAGES.get['create_unavailable_slot'])
```
Comment (ramprasadm-CBT) [inline backend/base/views.py]: why change to square brackets?

File: backend/base/views.py [hunk 2]
```diff
@@ -1557,7 +1424,7 @@ def post(self, request, *args, **kwargs):
             try:
                 slot = UnavailableSlot.objects.get(id=id)
                 slot.delete()
-                messages.success(self.request, 'Deleted Unavailable slot successfully !!!')
+                messages.success(self.request, SUCCESS_MESSAGES['delete_unavailable_slot'])
```
Comment (ramprasadm-CBT) [inline backend/base/views.py]: Why not .get and why square brackets?

File: backend/base/views.py [hunk 3]
```diff
@@ -1517,7 +1384,7 @@ def form_valid(self, form):
                 raise KeyError('Location PK not found in session')
             location = OrganizationLocation.objects.get(pk=pk)
             form.instance.location = location
-            messages.success(self.request, 'Created Additional slot successfully !!!')
+            messages.success(self.request, SUCCESS_MESSAGES.get['create_additional_slot'])
```
Comment (ramprasadm-CBT) [inline backend/base/views.py]: Square bracket reason?

File: backend/base/views.py [hunk 4]
```diff
@@ -1488,7 +1355,7 @@ def post(self, request, *args, **kwargs):
             try:
                 slot = AdditionalSlot.objects.get(id=id)
                 slot.delete()
-                messages.success(self.request, 'Deleted Additional slot successfully !!!')
+                messages.success(self.request, SUCCESS_MESSAGES.get['delete_additional_slot'])
```
Comment (ramprasadm-CBT) [inline backend/base/views.py]: change in syntax. reason?

File: backend/base/views.py [hunk 5]
```diff
@@ -1460,7 +1327,7 @@ def post(self, request, *args, **kwargs):
                     messages.error(request, 'Location not found.')
                     return redirect('error-url')
 
-            messages.success(request, 'Slots created successfully.')
+            messages.success(request, SUCCESS_MESSAGES.get['create_slot'])
```
Comment (ramprasadm-CBT) [inline backend/base/views.py]: syntax change in success message retrieval. why?

File: backend/base/views.py [hunk 6]
```diff
@@ -1631,9 +1496,10 @@ def update_working_days(request, location_pk):
         formset = OrganizationLocationWorkingDaysFormSet(request.POST, queryset=queryset)
         if formset.is_valid():
             formset.save()
-            return JsonResponse({'status': 'success', 'message': 'Working days updated successfully.'})
+            return JsonResponse({'status': 'success', 'message': SUCCESS_MESSAGES.get('update_workingdays')})
```
Comment (ramprasadm-CBT) [inline backend/base/views.py:1555]: changed back to usual syntax. follow one pattern.

File: backend/frontend/src/store.js [hunk 7]
```diff
@@ -90,7 +90,7 @@ const middleware = [thunk];
 const store = createStore(
   reducer,
   initialState,
-  composeWithDevTools((applyMiddleware(...middleware))),
```
Comment (ramprasadm-CBT) [inline backend/frontend/src/store.js:97]: comment it when you push. in local we can uncomment it for development needs.

File: backend/base/messages.py [hunk 8]
```diff
@@ -1,9 +1,52 @@
-# messages.py
+# messagespy
```
Comment (ramprasadm-CBT) [inline backend/base/messages.py]: Thats a commented out file name for my reference in note-pad. i forgot to remove it while pushing. No need for change in it.

**Review by ramprasadm-CBT — CHANGES_REQUESTED** [review]
Please change the mentioned below. Mainly syntax changes. Minor fix

File: backend/base/templates/slot-list.html [hunk 9]
```diff
@@ -224,6 +197,12 @@ <h6 class="mb-5">Available Slots are:</h6>
                 });
             }
         });
+        const $successAlert = $('#successAlert');
+        if (successAlert.length) {
+            setTimeout(() => {
+                successAlert.hide();
+            }, 2500);
+        }
     </script>
```
Comment (rugvedcbt) [inline backend/base/templates/slot-list.html]: check this function whether its working. constant is defined with $ symbol, but used without symbol inside function.

**Review by rugvedcbt — CHANGES_REQUESTED** [review]

**Review by ramprasadm-CBT — APPROVED** [review]

File: backend/base/templates/slot-list.html [hunk 10]
```diff
@@ -13,57 +13,22 @@
     <link href="https://use.fontawesome.com/releases/v6.3.0/css/all.css" rel="stylesheet">
     <link href="{% static 'styles.css' %}" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
-    <!-- <style>
-        .location-card {
-            display: flex;
-            justify-content: space-between;
-            align-items: center;
+     <style>
```
Comment (rugvedcbt) [inline backend/base/templates/slot-list.html]: external css file can be used

File: backend/base/templates/slot_location.html [hunk 11]
```diff
@@ -30,14 +30,13 @@
...
-            height: 60px;
-        }
-
+            right: 20px;
+            background-color: #323231;
+            color: white;
+            padding: 10px 20px;
+            border-radius: 5px;
+            z-index: 1050;
+            }
```
Comment (rugvedcbt) [inline backend/base/templates/slot_location.html]: can use external css files.

File: backend/base/templates/temp-slots-list.html [hunk 12]
```diff
@@ -16,86 +16,45 @@
...
+            background-color: #323231;
+            color: white;
+            padding: 10px 20px;
+            border-radius: 5px;
+            z-index: 1050;
+        }
+        #successAlert p {
+            text-align: center;
+            margin: 0;
+        }
```
Comment (rugvedcbt) [inline backend/base/templates/temp-slots-list.html]: can use external css files.

File: backend/base/templates/temp-slots-list.html [hunk 13]
```diff
@@ -16,86 +16,45 @@
 
     <style>
         .popup {
```
Comment (rugvedcbt) [inline backend/base/templates/temp-slots-list.html]: can use external css files.

File: backend/base/templates/unavailable-slot-list.html [hunk 14]
```diff
@@ -16,86 +16,44 @@
 
     <style>
         .popup {
```
Comment (rugvedcbt) [inline backend/base/templates/unavailable-slot-list.html]: can use external css files.

File: backend/base/templates/update_slot.html [hunk 15]
```diff
@@ -106,9 +103,7 @@
           background-color: rgba(0, 0, 0, 0.5);
           z-index: 9998;
       }
-        /* .datatable-top {
-            visibility: hidden;
-        } */
+
```
Comment (rugvedcbt) [inline backend/base/templates/update_slot.html]: can use external css files.

**Review by rugvedcbt — COMMENTED** [review]
can use external css files instead of internal css in html wherever its possible.

File: backend/base/static/css/slot-list.css [hunk 16]
```diff
@@ -0,0 +1,14 @@
...
+  position: fixed;
+  top: 70px;
+  right: 20px;
+  background-color: #323231;
+  color: white;
+  padding: 10px 20px;
+  border-radius: 5px;
+  z-index: 1050;
+}
+#successAlert p {
```
Comment (murucbt) [inline backend/base/static/css/slot-list.css]: Can you try like below ?

```
#successAlert {
  position: fixed;
  top: 70px;
  right: 20px;
  background-color: #323231;
  color: white;
  padding: 10px 20px;
  border-radius: 5px;
  z-index: 1050;

  p {
    text-align: center;
    margin: 0;
  }
}
```

File: backend/base/messages.py [hunk 17]
```diff
@@ -1,9 +1,53 @@
...
+    'delete_image': 'Image deleted successfully',
+    'delete_court': 'Court deleted successfully',
+    'delete_slot': 'Slot deleted successfully',
+    'delete_additional_slot' : 'Deleted Additional slot successfully',
+    'delete_unavailable_slot' : 'Deleted Unavailable slot successfully',
+
+    #org
+    'preview' : 'Preview submitted successfully',
+    'org_status' : 'Organization status updated sucessfully',
+    'orglocation_status' : 'Organization location status updated sucessfully',
```
Comment (murucbt) [inline backend/base/messages.py]: Seems typo. Check other places too.

**Review by murucbt — COMMENTED** [review]
Small Nits.

File: backend/base/static/css/update_slot.css [hunk 18]
```diff
@@ -0,0 +1,84 @@
...
+  border: none;
+  border-radius: 5px;
+  color: white;
+  background: #007bff;
+  text-align: center;
+  font-size: 16px;
+}
+.popup h2 {
+  margin-bottom: 15px;
+}
```
Comment (rugvedcbt) [inline backend/base/static/css/update_slot.css]: .popup {
  display: none;
  position: absolute;
  top: 50%;
  left: 50%;
  transform: translate(-19%, -27%);
  background-color: #fff;
  padding: 40px 40px;
  border: 2px solid #888;
  border-radius: 5px;
  z-index: 9999;
  button {
  width: 25%;
  padding: 5px 10px;
  border: none;
  border-radius: 5px;
  color: white;
  background: #007bff;
  text-align: center;
  font-size: 16px;
}
h2 {
  margin-bottom: 15px;
}}
can use css like this

**Review by rugvedcbt — APPROVED** [review]

File: backend/base/static/css/update_slot.css (hunk 18 above)
Comment (TharunCBT) [reply backend/base/static/css/update_slot.css]: Changed it!!

**Pull request merged**